__tests__/app.test.js

```diff
@@ -409,5 +409,26 @@ describe('DELETE /api/comments/:comment_id', () => {
             expect(response.body.msg).toBe('bad request');
         });
     });
+});
+
+
+describe('GET /api/users', () => {
+    test('returns array of objects with correct properties', () => {
```

Comment:
I would prefer to see this be more specific, what is the array of objects? This can simply say an array or a list of users.